.github/workflows/backport.yml

```diff
@@ -0,0 +1,24 @@
+name: Backport
+on:
+  pull_request_target:
+    types:
+      - closed
+      - labeled
+
+jobs:
+  backport:
+    name: Backport
+    runs-on: ubuntu-latest
+    if: >
+      github.event.pull_request.merged
+      && (
+        github.event.action == 'closed'
```

Comment:
Is this condition required here? If `merged` is true, then it was closed. Also, the `||` leads me to think it will apply to more PRs than only those with the tag.

Reply:
It's tricky because we have to handle multiple labels on the same PR.

Very rough pseudo code
```
if merged {
   if closed { return true }
   if labelled { return true }
}
return false
```

Reply:
Shouldn’t it be only 
```
if merged {
   if labelled { return true }
}
return false
```

Reply:
no because that won't catch PR's that were, opened, then labelled, then closed